src/constants/index.ts

```diff
@@ -6,3 +6,5 @@ export enum ToggleSwitchLabelPositioningValues {
 }
 
 export const TooltipDefaultPlacement = 'top';
+
+export const PrivateChannelType = 'P';
```

Comment:
Why did you add this again and not using `mattermost-redux`?

Reply:
Added to-do to use `mattermost-redux` as discussed.